wcfsetup/install/lang/en.xml

```diff
@@ -802,13 +802,13 @@ Examples for medium ID detection:
 		<item name="wcf.acp.option.session_enable_virtualization.description"><![CDATA[Users can log-in from multiple devices (PC, tablet, smartphone, etc) concurrently, all devices will share the same session. Disabling this option will restrict users to be logged-in with only one device at once, causing every login to terminate sessions used by the user’s other devices. It is not recommended to disable this option and you should be well aware of the consequences if you decide to disable it.]]></item>
 		<item name="wcf.acp.option.show_clock"><![CDATA[Show current time in page footer]]></item>
 		<item name="wcf.acp.option.timezone"><![CDATA[Timezone]]></item>
-		<item name="wcf.acp.option.timezone.description"><![CDATA[Default timezone of your page]]></item>
-		<item name="wcf.acp.option.user_online_timeout"><![CDATA[User-Online timeout]]></item>
+		<item name="wcf.acp.option.timezone.description"><![CDATA[The default timezone of your page.]]></item>
+		<item name="wcf.acp.option.user_online_timeout"><![CDATA["Users Online" Timeout]]></item>
```

Comment:
Please use the proper unicode quotes: 

http://www.fileformat.info/info/unicode/char/201c/index.htm
http://www.fileformat.info/info/unicode/char/201d/index.htm